### PR TITLE
feat: Deploy Eclipse Che on AKS

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
feat: Deploy Eclipse Che on AKS

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/22665

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Microsoft Azure is a cloud computing service created by Microsoft for building, testing, deploying, and managing applications and services through Microsoft-managed data centers.
+
+This section provides information about installing, enabling, and basic use of {prod-short} on Microsoft Azure.
+
+.Prerequisites
+
+* `helm` the package manager for {kubernetes}. See: link:https://helm.sh/docs/intro/install/[Installing Helm].
+
+* `az` the Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install the Azure CLI].
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
* `az` the Microsoft Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install Microsoft Azure CLI].
```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
++
+[source,shell,subs="attributes+"]
+----
+az network dns record-set a add-record \
+  --resource-group $ECLIPSE_CHE_RESOURCE_GROUP \
+  --zone-name $DOMAIN_NAME \
+  --record-set-name "*" \
+  --ipv4-address $({orch-cli} get service -n ingress-nginx ingress-nginx-controller -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+IMPORTANT: If you use a registrar like GoDaddy, you will need to add
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc]:
```suggestion
IMPORTANT: If you use a registrar such as GoDaddy, you will need to add
```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+two DNS records in your registrar of type `A`, names `@` and `*` and point them to the
+IP address of the ingress controller.
++
+.Additional resources
+
+* link:https://learn.microsoft.com/en-us/azure/dns/dns-getstarted-cli[Create an Azure DNS zone and record using Azure CLI]
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc]:
```suggestion
* link:https://learn.microsoft.com/en-us/azure/dns/dns-getstarted-cli[Create a Microsoft Azure DNS zone and record using Microsoft Azure CLI]
```

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  - '*.$DOMAIN_NAME'
+  usages:
+    - server auth
+    - digital signature
+    - key encipherment
+    - key agreement
+    - data encipherment
+EOF
+----
+IMPORTANT: If you use a registrar like GoDaddy, you will need to duplicate DNS records in your registrar of
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc]:
```suggestion
IMPORTANT: If you use a registrar such as GoDaddy, you will need to duplicate DNS records in your registrar of
```

File: modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc [hunk 5]
```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-che-on-microsoft-azure-kubernetes-service"]
+= Installing {prod-short} on Microsoft Azure Kubernetes Service
+
+.Procedure
+
+. Prepare a CheCluster patch yaml file:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
. Prepare a CheCluster patch YAML file:
```

File: modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
++
+[source,shell]
+----
+az aks get-credentials \
+  --resource-group $ECLIPSE_CHE_RESOURCE_GROUP \
+  --name $AKS_CLUSTER_NAME \
+  --admin
+----
+
+. Set `kubelogin` to use the Azure CLI:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc]:
```suggestion
. Set `kubelogin` to use the Microsoft Azure CLI:
```

**Review by deerskindoll — COMMENTED** [review]
let's get rid of the vale warnings first so that we can generate a preview of the page

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 7]
```diff
@@ -0,0 +1,21 @@
+:_content-type: PROCEDURE
+:description: Installing {prod-short} in the Cloud
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
:description: Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+:keywords: overview, running-che-in-the-cloud, installing-che-in-the-cloud
+:navtitle: Installing {prod-short} in the Cloud
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
:navtitle: Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="installing-{prod-id-short}-locally"]
+= Installing {prod-short} in the Cloud
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
= Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+This section contains instructions for deploying and running {prod} in the Cloud.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
Deploy and run {prod} in the cloud.
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+.Prerequisites
+
+* A {platforms-name} cluster to deploy {prod-short} on.
+
+* `{prod-cli}` the command line tool for {prod}. See: xref:installing-the-chectl-management-tool.adoc[].
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
* `{prod-cli}`: The command line tool for {prod}. See: xref:installing-the-chectl-management-tool.adoc[].
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+== Deploying {prod-short} in the Cloud
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
== Deploying {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+Choose one of the following procedures to start the {prod-short} Server in the Cloud using the `{prod-cli}` tool.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc]:
```suggestion
Follow the instructions below to start the {prod-short} Server in the cloud using the `{prod-cli}` tool.
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+[id="installing-{prod-id-short}-on-microsoft-azure"]
+= Installing {prod-short} on Microsoft Azure
+
+Microsoft Azure is a cloud computing service created by Microsoft for building, testing, deploying, and managing applications and services through Microsoft-managed data centers.
+
+This section provides information about installing, enabling, and basic use of {prod-short} on Microsoft Azure.
+
+.Prerequisites
+
+* `helm` the package manager for {kubernetes}. See: link:https://helm.sh/docs/intro/install/[Installing Helm].
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc:15]:
```suggestion
* `helm`: The package manager for {kubernetes}. See: link:https://helm.sh/docs/intro/install/[Installing Helm].
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+
+* `az` the Microsoft Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install Microsoft Azure CLI].
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc:17]:
```suggestion
* `az`: The Microsoft Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install Microsoft Azure CLI].
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 7 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:2]:
```suggestion
:description: Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 8 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:4]:
```suggestion
:navtitle: Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 9 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:7]:
```suggestion
= Installing {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 10 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:9]:
```suggestion
Deploy and run {prod} in the cloud.
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 11 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:15]:
```suggestion
* `{prod-cli}`: The command line tool for {prod}. See: xref:installing-the-chectl-management-tool.adoc[].
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 12 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:17]:
```suggestion
== Deploying {prod-short} in the cloud
```

File: modules/administration-guide/pages/installing-che-in-the-cloud.adoc (hunk 13 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-in-the-cloud.adoc:19]:
```suggestion
Follow the instructions below to start the {prod-short} Server in the cloud using the `{prod-cli}` tool.
```

**Review by deerskindoll — COMMENTED** [review]
edited installing-che-in-the-cloud.adoc

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 16]
```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+:description: Installing {prod-short} on Microsoft Azure
+:keywords: overview, installing-che-on-microsoft-azure
+:navtitle: Installing {prod-short} on Microsoft Azure
+
+[id="installing-{prod-id-short}-on-microsoft-azure"]
+= Installing {prod-short} on Microsoft Azure
+
+Microsoft Azure is a cloud computing service created by Microsoft for building, testing, deploying, and managing applications and services through Microsoft-managed data centers.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
Microsoft Azure is a cloud computing service for building, testing, deploying, and managing applications and services through Microsoft-managed data centers.
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+This section provides information about installing, enabling, and basic use of {prod-short} on Microsoft Azure.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
Follow the instructions below to install and enable {prod-short} on Microsoft Azure.
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+
+.Prerequisites
+
+* `helm` the package manager for {kubernetes}. See: link:https://helm.sh/docs/intro/install/[Installing Helm].
+
+* `az` the Microsoft Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install Microsoft Azure CLI].
+
+* `kubelogin` the credential plugins See: link:https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli[How to install kubelogin].
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
* `helm`: The package manager for {kubernetes}. See: link:https://helm.sh/docs/intro/install/[Installing Helm].

* `az`:  The Microsoft Azure CLI command line tool. See: link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[How to install Microsoft Azure CLI].

* `kubelogin`: The credential plugin. See: link:https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli[How to install kubelogin].
```

**Review by deerskindoll — COMMENTED** [review]
edited installing-che-on-microsoft-azure.adoc

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 19]
```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+:description: Installing {prod-short} on Microsoft Azure
+:keywords: overview, installing-che-on-microsoft-azure
+:navtitle: Installing {prod-short} on Microsoft Azure
+
+[id="installing-{prod-id-short}-on-microsoft-azure"]
+= Installing {prod-short} on Microsoft Azure
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]: based on the contents of the assembly, would it make sense to change the title to `Installing and setting up {prod-short} on Microsoft Azure`?

Comment (tolusha) [reply modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]: @deerskindoll 
Yes, it make sense.

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 20]
```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+:description: Installing {prod-short} on Microsoft Azure
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
:description: Installing and setting up {prod-short} on Microsoft Azure
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+:keywords: overview, installing-che-on-microsoft-azure
+:navtitle: Installing {prod-short} on Microsoft Azure
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
:navtitle: Installing and setting up {prod-short} on Microsoft Azure
```

File: modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc (hunk 19 above)
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc]:
```suggestion
= Installing and setting up {prod-short} on Microsoft Azure
```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 22]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="configuring-DNS-on-microsoft-azure"]
+= Configuring DNS on Microsoft Azure
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc:7]:
```suggestion

Configure DNS on Microsoft Azure. Before you start, make sure you have a registered domain.

.Prerequisites

* A registered domain.

```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+.Procedure
+
+. Define the domain name. You will need to register a domain unless you already have one.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc]:
```suggestion
. Define the domain name.
```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 24]
```diff
@@ -0,0 +1,42 @@
...
+----
+az network dns record-set a add-record \
+  --resource-group $ECLIPSE_CHE_RESOURCE_GROUP \
+  --zone-name $DOMAIN_NAME \
+  --record-set-name "*" \
+  --ipv4-address $({orch-cli} get service -n ingress-nginx ingress-nginx-controller -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+IMPORTANT: If you use a registrar such as GoDaddy, you will need to add
+two DNS records in your registrar of type `A`, names `@` and `*` and point them to the
+IP address of the ingress controller.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc]:
```suggestion
[IMPORTANT]
====
If you use a registrar such as GoDaddy, you will need to add
two DNS records in your registrar of type `A`, names `@` and `*`, and point them to the
IP address of the ingress controller.
====
```

**Review by deerskindoll — COMMENTED** [review]
edited configuring-dns-on-microsoft-azure.adoc

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 25]
```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="creating-lets-encrypt-certificate-for-{prod-id-short}-on-microsoft-azure"]
+= Creating Let's encrypt certificate for {prod-id-short} on Microsoft Azure
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc:9]:
```suggestion

Follow these instructions to create a Let's Encrypt certificate for {prod-short} on Microsoft Azure.

```

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 26]
```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="creating-lets-encrypt-certificate-for-{prod-id-short}-on-microsoft-azure"]
+= Creating Let's encrypt certificate for {prod-id-short} on Microsoft Azure
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc]:
```suggestion
= Creating Let's Encrypt certificate for {prod-id-short} on Microsoft Azure
```

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 27]
```diff
@@ -0,0 +1,108 @@
...
+----
+
+. Create the {prod-namespace} namespace:
++
+[source,shell,subs="attributes+"]
+----
+{orch-cli} create namespace {prod-namespace}
+----
+
+. Creating a Service Account Secret:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc]:
```suggestion
. Create a Service Account Secret:
```

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 28]
```diff
@@ -0,0 +1,108 @@
...
++
+[source,shell,subs="attributes+"]
+----
+{orch-cli} create secret generic azuredns-config \
+  --from-literal=clientSecret=$(az ad sp create-for-rbac --name $CERT_MANAGER_SERVICE_PRINCIPAL_NAME --query "password" --output tsv) \
+  --namespace {prod-namespace}
+----
+
+. Create the Issuer. Replace `MY_EMAIL_ADDRESS` with a valid address:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc]:
```suggestion
. Create the Issuer and replace `MY_EMAIL_ADDRESS` with a valid address:
```

File: modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc [hunk 29]
```diff
@@ -0,0 +1,108 @@
...
+  usages:
+    - server auth
+    - digital signature
+    - key encipherment
+    - key agreement
+    - data encipherment
+EOF
+----
+IMPORTANT: If you use a registrar such as GoDaddy, you will need to duplicate DNS records in your registrar of
+type `TXT` and name `_acme-challenge`.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_creating-lets-encrypt-certificate-for-che-on-microsoft-azure.adoc]:
```suggestion
[IMPORTANT]
====
If you use a registrar such as GoDaddy, you need to duplicate the following DNS records in your registrar:
* type: `TXT`
* name: `_acme-challenge`.
====
```

File: modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc [hunk 30]
```diff
@@ -0,0 +1,51 @@
...
+  --resource-group $ECLIPSE_CHE_RESOURCE_GROUP \
+  --zone-name $DOMAIN_NAME \
+  --record-set-name "*" \
+  --ipv4-address $({orch-cli} get service -n ingress-nginx ingress-nginx-controller -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+----
+[IMPORTANT]
+====
+If you use a registrar such as GoDaddy, you will need to add
+two DNS records in your registrar of type `A`, names `@` and `*`, and point them to the
+IP address of the ingress controller.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_configuring-dns-on-microsoft-azure.adoc]:
```suggestion
If you use a registrar such as GoDaddy, you will need to add
the following two DNS records in your registrar and point them to the
IP address of the ingress controller:
* type: `A`
* names: `@` and `*`
```

**Review by deerskindoll — COMMENTED** [review]
edited creating-lets-encrypt(...).adoc

File: modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc [hunk 31]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-cert-manager-on-microsoft-azure-kubernetes-service"]
+= Installing cert-manager on Microsoft Azure Kubernetes Service
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc:9]:
```suggestion

Learn how to install cert-manager on Microsoft Azure Kubernetes Service.

```

File: modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+.Procedure
+
+. To install the cert-manager:
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
. Install the cert-manager:
```

File: modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc [hunk 33]
```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-che-on-microsoft-azure-kubernetes-service"]
+= Installing {prod-short} on Microsoft Azure Kubernetes Service
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc:9]:
```suggestion

Install {prod-short} on Microsoft Azure Kubernetes Service.

```

**Review by deerskindoll — COMMENTED** [review]
edited installing-cert-manager(...).adoc and installing-che(...).adoc

File: modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc [hunk 34]
```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service"]
+= Installing NGINX Ingress Controller on Microsoft Azure Kubernetes Service
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc:9]:
```suggestion

Use the following instructions to install the NGINX Ingress Controller on Microsoft Azure Kubernetes Service.

```

File: modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc [hunk 35]
```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="preparing-microsoft-azure-for-installing-che"]
+= Preparing Microsoft Azure for installing {prod-short}
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc:9]:
```suggestion

Prepare Microsoft Azure for {prod-short} installation.

```

File: modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc [hunk 36]
```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="preparing-microsoft-azure-for-installing-che"]
+= Preparing Microsoft Azure for installing {prod-short}
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc]:
```suggestion
= Preparing Microsoft Azure for {prod-short} installation
```

File: modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc [hunk 37]
```diff
@@ -0,0 +1,91 @@
...
+----
+
+. View the pods in the cluster :
++
+[source,shell,subs="+attributes"]
+----
+{orch-cli} get pods --all-namespaces
+----
++
+All the Pods in the running state are displayed.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_preparing-microsoft-azure-for-installing-che.adoc]:
```suggestion
. Verification

All pods in the running state are displayed.
```

**Review by deerskindoll — COMMENTED** [review]
edited installing-nginx(...).adoc and preparing-microsoft-azure(...).adoc

File: modules/administration-guide/partials/proc_registering-application-on-microsoft-azure.adoc [hunk 38]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="registering-the-application-on-microsoft-azure-ad-portal"]
+= Registering a client application in Microsoft Entra ID
+
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_registering-application-on-microsoft-azure.adoc:7]:
```suggestion

Learn how to register a client application in Microsoft Entra ID

```

**Review by deerskindoll — COMMENTED** [review]
edited registering-application(...).adoc

this should be the last suggestion :D

File: modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc [hunk 39]
```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-cert-manager-on-microsoft-azure-kubernetes-service"]
+= Installing cert-manager on Microsoft Azure Kubernetes Service
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
= Installing cert-manager on Microsoft Azure {kubernetes} Service
```

File: modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc [hunk 40]
```diff
@@ -0,0 +1,28 @@
...
+
+Learn how to install cert-manager on Microsoft Azure Kubernetes Service.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-cert-manager-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
Learn how to install cert-manager on Microsoft Azure {kubernetes} Service.
```

File: modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc [hunk 41]
```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-che-on-microsoft-azure-kubernetes-service"]
+= Installing {prod-short} on Microsoft Azure Kubernetes Service
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
= Installing {prod-short} on Microsoft Azure {kubernetes} Service
```

File: modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc [hunk 42]
```diff
@@ -0,0 +1,62 @@
...
+
+Install {prod-short} on Microsoft Azure Kubernetes Service.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-che-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
Install {prod-short} on Microsoft Azure {kubernetes} Service.
```

File: modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc [hunk 43]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-microsoft-azure
+
+[id="installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service"]
+= Installing NGINX Ingress Controller on Microsoft Azure Kubernetes Service
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
= Installing NGINX Ingress Controller on Microsoft Azure {kubernetes} Service
```

File: modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc [hunk 44]
```diff
@@ -0,0 +1,37 @@
...
+
+Use the following instructions to install the NGINX Ingress Controller on Microsoft Azure Kubernetes Service.
```
Comment (deerskindoll) [inline modules/administration-guide/partials/proc_installing-nginx-ingress-controller-on-microsoft-azure-kubernetes-service.adoc]:
```suggestion
Use the following instructions to install the NGINX Ingress Controller on Microsoft Azure {kubernetes} Service.
```

**Review by deerskindoll — APPROVED** [review]
fixed a new vale error, otherwise lgtm

**Pull request merged**